README.md

```diff
@@ -8,15 +8,23 @@ An operator to perform lifecycle management (install/upgrade/uninstall) of [Kata
 
 1. Make sure that `oc` is configured to talk to the cluster
 
-2. Run to install the Kata Operator: 
+2. Clone the Kata Operator repository and check out the branch matching with the Openshift version. e.g. If you are running Openshift 4.7 then,
+
+
+   ```
+   git clone https://github.com/openshift/kata-operator 
+   git checkout -b release-4.7 --track origin/release-4.7
+   ```
+3. Install the Kata Operator on the cluster,
 
    ```
-   curl https://raw.githubusercontent.com/openshift/kata-operator/master/deploy/deploy.sh | bash
+   make install && make install && make deploy IMG=quay.io/harpatil/kata-operator:0.4
```

Comment:
Keeping it pinned to a personal image until we have the image deployed in official repository